### ISSUE TITLE
Staging Review finding: Download links cannot be reached using keyboard

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#104904]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Iterate, Innovate, & Run (IIR)
**Product:** 1095-B Tax Form
**Feature:** Digital Access MVP

## Findings details
**VA.gov Experience Standard - issue:** [User can complete necessary flows.](https://design.va.gov/about/experience-standards/#user-can-complete-necessary-flows)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

Neither the "Download PDF" nor "Download Text file" links can receive keyboard focus, meaning keyboard-only users won't be able to download them.

## Recommended action

From your testing artifact, I see that you already know about this one. By convention, links require an `href` attribute in order for browsers to add them to the tab order. A perfunctory `href="#"` is sufficient for something like this, as would be a data URL.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 2.1.1 Keyboard - Level A](https://www.w3.org/TR/WCAG21/#keyboard)
- **Modality:** Keyboard
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [x] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by mcommons10** [comment]
This has been fixed and was addressed in this ticket: https://github.com/department-of-veterans-affairs/va-iir/issues/1471
